ui/app/components/app/connected-accounts-list/index.scss

```
@@ -0,0 +1,155 @@
+.connected-accounts-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__identicon {
+    margin-right: 8px;
+  }
+
+  &__account-name {
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 20px;
+  }
+
+  &__primary {
+    font-style: italic;
+  }
+
+  &__account-status {
+    color: #6A737D;
```

Nit: We should use variables where they're already defined

```suggestion
    color: $Grey-500;
```

---

The same goes for the other colours as well - many of them are declared in variables already.